runtime/src/genesis_utils.rs

```diff
@@ -10,11 +11,15 @@ use solana_sdk::{
     system_program,
 };
 use solana_stake_program::stake_state;
+use solana_stake_program::stake_state::StakeState;
 use solana_vote_program::vote_state;
 use std::borrow::Borrow;
 
 // The default stake placed with the bootstrap validator
-pub const BOOTSTRAP_VALIDATOR_LAMPORTS: u64 = 42;
+// fun fact: we're very close to make this const fn.
+pub fn bootstrap_validator_lamports() -> u64 {
+    StakeState::get_rent_exempt_reserve(&Rent::default())
+}
```

Comment:
I actually find this redefinition a little confusing, because it is only when `bootstrap_validator_stake_lamports` is too low that we will see the problem described in this PR.
A low `bootstrap_validator_lamports` may cause different problems (which I know intimately after https://github.com/solana-labs/solana/pull/13627 :) ); if it's tied to the rent-exempt-balance of the account I think we will be more prone to misuse.

What about defining this separately as `bootstrap_validator_stake_lamports()` and only replacing the stake-lamports values?

Reply:
thanks for great suggestion. Well, I didn't pay enough attention to the code... I think I've applied what you have suggested. :)